bindings/python/test/test_arrow.py

```diff
@@ -147,3 +153,24 @@ def test_aggregate_omits_id_if_not_in_schema(self):
             if op_name == '$project':
                 self.assertFalse(stage[op_name]['_id'])
                 break
+
+
+class TestArrowExplicitApi(TestArrowApiMixin, unittest.TestCase):
```

Comment:
Isn't it overkill to run all the test cases on both the explicit and patched api given that they are the exact same method? I think it would make more sense just to run all the tests with the explicit api and then have one test for the patched methods (or vice versa).

Reply:
Yes, it kind of is overkill but honestly, the cost is so low right now... It takes <1 second to run this entire test file on my laptop...

Reply:
Okay, let's keep this in mind for future testing.